hydpy/models/hland_v1.py

```diff
@@ -139,6 +140,19 @@
 ...         altitudefactor(-0.1)
 ...         precipitationfactor(0.1)
 
+|hland_v1| uses an optional submodel for calculating the runoff concentration. Therefore,
+we select |rconc_uh|:
+
+>>> with model.add_rconcmodel_v1("rconc_uh"):
+...     uh("triangle", tb=3.0)
+
+The submodel has its own function to return the submodel waterbalance. As an argument,
+it needs the initial submodel conditions. If the conditions are empty, the returned
+water balance is zero:
+
+>>> print(model.get_rconcmodel_waterbalance([]))
+0.0
+
```

Comment:
Confusing. Do you mean, "The model has a function to return the submodel's water balance?" Or "The submodel has a function to return its water balance"?

Either way, if such explanations are necessary, we should likely move them to the docstring of  `get_rconcmodel_waterbalance` or `get_waterbalance`.

Also, it is unclear why the returned value is zero. Shouldn't the method raise an error when the input data is incomplete?

Reply:
As the method get_rconcmodel_waterbalance is covered already in the integration test of hland_v1 when checking the water balance of the application model, I removed these lines. The reason for returning 0.0 if there are no initial conditions was that the submdel is optional. But, as we test for the existence of rconcmodel before calling get_rconcmodel_waterbalance, now I implemented the raise of a ValueError when initial conditions are empty.